content/doc/book/pipeline/multibranch.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:imagesdir: /doc/book/resources/pipeline
+:hide-uri-scheme:
+:toc:
+
+= Multibranch Pipelines
+
+In the <<jenkinsfile#, previous section>> a `Jenkinsfile` which could be
+checked into source control was implemented. This section covers the concept of
+*Multibranch* Pipelines which build on the `Jenkinsfile` foundation to provide
```

Comment:
Active voice is fine for me too actually, it's the pronouns I abhor.

My brain is still sufficiently fried from this documentation, that I would suggest you take a pass through this at some point if you wish to change over to active voice.